lib/bolt/shell/powershell/snippets.rb

```diff
@@ -48,6 +48,7 @@ def run_script(arguments, script_path)
             build_arg_list = arguments.map do |a|
               "$invokeArgs.ArgumentList += @'\n#{a}\n'@"
             end.join("\n")
+
```

Comment:
This seems incidental - would you be ok resetting just this file to keep the Git history more navigable?